docs/5.0/enterprise/quickstart-enterprise.md

```diff
@@ -178,34 +178,50 @@ The output will look like this (re-formatted here to use compact YAML
 representation for brevity):
 
 ```yaml
+
 kind: role
-version: v3
 metadata:
   name: admin
 spec:
-  options:
-    cert_format: standard
-    forward_agent: true
-    max_session_ttl: 30h0m0s
-    port_forwarding: true
-  # allow rules:
   allow:
     logins:
     - '{% raw %}{{internal.logins}}{% endraw %}'
     - root
     node_labels:
       '*': '*'
+    app_labels:
+      '*': '*'
+    kubernetes_groups:
+    - '{% raw %}{{internal.kubernetes_groups}}{% endraw %}'
```

Comment:
I don't think this `{% raw %}` syntax is needed any more with the new docs engine, you can just put the values in as-is.